### ISSUE TITLE
Sensor tower should obscure some xeno stuff

### ISSUE THREAD

**Issue by Vermidia** [RMC-14/RMC-14#5197]
## Description

All xeno structures shouldn't show to the marines Thats it.

**Reproduction**

**Screenshots**

**Additional context**

**Comment by Boaz1111** [comment]
so no tunnels visible?

**Comment by Vermidia** [comment]
yes it only shows living xenos to the marines, and all with the same icon
